lsp/src/handle.rs

```diff
@@ -60,6 +81,30 @@ fn handle_didChange(noti: Notification) -> Option<HtmxResult> {
     return None;
 }
 
+#[allow(non_snake_case)]
+fn handle_didOpen(noti: Notification) -> Option<HtmxResult> {
```

Comment:
Since this feature doesn't require editing the document, it needs to read the current document and store once it is opened